### PR TITLE
Fix for #1

### PR DESCRIPTION
**Pull request by danielleberre**
The links to the examples are incomplete as mentioned in #1 . This PR fixes the links to see the examples in github.io

### CODE REVIEW

File: snowcampio-2018-01-26.md [hunk 1]
```diff
@@ -1093,7 +1093,7 @@ A development pattern
 
 **Docker Image** dfd
 
-**Source code**<br>https://github/l0rd/containerspatterns/dfd
+**Source code**<br>https://l0rd.github.io/containerspatterns/dfd
```
Comment (dolmen) [inline snowcampio-2018-01-26.md]: https://l0rd.github.io/containerspatterns/dfd => 404 :(

File: snowcampio-2018-01-26.md [hunk 2]
```diff
@@ -1028,7 +1028,7 @@ A development pattern
 
 **Docker Image**<br>obi-java
 
-**Source code**<br>https://github/l0rd/containerspatterns/OBI
+**Source code**<br>https://l0rd.github.io/containerspatterns/OBI
```
Comment (dolmen) [inline snowcampio-2018-01-26.md]: https://l0rd.github.io/containerspatterns/OBI => 404 :(

File: snowcampio-2018-01-26.md [hunk 3]
```diff
@@ -956,7 +956,7 @@ A runtime pattern
 
 **Docker Image**<br>httpd
 
-**Source code**<br>https://github/l0rd/containerspatterns/EFE/
+**Source code**<br>https://l0rd.github.io/containerspatterns/EFE/
```
Comment (dolmen) [inline snowcampio-2018-01-26.md]: https://l0rd.github.io/containerspatterns/EFE/ => 404 :(

File: snowcampio-2018-01-26.md [hunk 4]
```diff
@@ -888,7 +888,7 @@ A runtime pattern
 
 **Docker Image**<br>ecc
 
-**Source code**<br>https://github/l0rd/containerspatterns/ECC
+**Source code**<br>https://l0rd.github.io/containerspatterns/ECC
```
Comment (dolmen) [inline snowcampio-2018-01-26.md]: https://l0rd.github.io/containerspatterns/ECC => 404 :(

File: snowcampio-2018-01-26.md [hunk 5]
```diff
@@ -176,7 +176,7 @@ template: inverse
 
 **Docker Image**<br>maven:3.3.3-jdk-8
 
-**Source code**<br>https://github/l0rd/containerspatterns/DYT
+**Source code**<br>https://l0rd.github.io/containerspatterns/DYT
```
Comment (dolmen) [inline snowcampio-2018-01-26.md]: https://l0rd.github.io/containerspatterns/DYT => 404 :(

File: snowcampio-2018-01-26.md [hunk 6]
```diff
@@ -247,7 +247,7 @@ template: inverse
 
 **Docker Image**<br>containerslanguages/rust-launcher
 
-**Source code**<br>https://github/l0rd/containerspatterns/CL
+**Source code**<br>https://l0rd.github.io/containerspatterns/CL
```
Comment (dolmen) [inline snowcampio-2018-01-26.md]: The final URL is https://l0rd.github.io/containerspatterns/CL/

File: snowcampio-2018-01-26.md [hunk 7]
```diff
@@ -341,7 +341,7 @@ template: inverse
 
 **Docker Image**<br>alpine
 
-**Source code**<br>https://github/l0rd/containerspatterns/HS
+**Source code**<br>https://l0rd.github.io/containerspatterns/HS
```
Comment (dolmen) [inline snowcampio-2018-01-26.md]: https://l0rd.github.io/containerspatterns/HS => 404 :(

File: snowcampio-2018-01-26.md [hunk 8]
```diff
@@ -410,7 +410,7 @@ template: inverse
 - maven:3.5-jdk-8 (build)
 - openjdk:8-jre (run)
 
-**Source code**<br>https://github/l0rd/containerspatterns/S2I
+**Source code**<br>https://l0rd.github.io/containerspatterns/S2I
```
Comment (dolmen) [inline snowcampio-2018-01-26.md]: The final URL is https://l0rd.github.io/containerspatterns/S2I/

File: snowcampio-2018-01-26.md [hunk 9]
```diff
@@ -477,7 +477,7 @@ template: inverse
 **Docker Images**
 httpd and ubuntu
 
-**Source code**<br>https://github/l0rd/containerspatterns/SC
+**Source code**<br>https://l0rd.github.io/containerspatterns/SC
```
Comment (dolmen) [inline snowcampio-2018-01-26.md]: The final URL is https://l0rd.github.io/containerspatterns/SC/

File: snowcampio-2018-01-26.md [hunk 10]
```diff
@@ -754,7 +754,7 @@ A runtime pattern
 
 **Docker Image**<br>containerslanguages/golang
 
-**Source code**<br>https://github/l0rd/containerspatterns/DSM
+**Source code**<br>https://l0rd.github.io/containerspatterns/DSM
```
Comment (dolmen) [inline snowcampio-2018-01-26.md]: The final URL is https://l0rd.github.io/containerspatterns/DSM/

File: snowcampio-2018-01-26.md [hunk 11]
```diff
@@ -676,7 +676,7 @@ A development pattern
 
 **Docker Image**<br>maven:3.3.3-jdk-8
 
-**Source code**<br>https://github/l0rd/containerspatterns/DYT
+**Source code**<br>https://l0rd.github.io/containerspatterns/DYT
```
Comment (dolmen) [inline snowcampio-2018-01-26.md]: https://l0rd.github.io/containerspatterns/DYT => 404 :(

File: snowcampio-2018-01-26.md [hunk 12]
```diff
@@ -1093,7 +1093,7 @@ A development pattern
 
 **Docker Image** dfd
 
-**Source code**<br>https://github/l0rd/containerspatterns/dfd
+**Source code**<br>https://l0rd.github.io/containerspatterns/dfd
 <br>https://github/polyfunc/flask-todolist
```
Comment (dolmen) [inline snowcampio-2018-01-26.md]: This link is incorrect too.

**Review by dolmen — CHANGES_REQUESTED** [review]
Most "fixed" links aren't: 404 :(

**Comment by danielleberre** [comment]
@dolmen I used the full github.com links when the github.io link does not work.

Not sure the links are expected to be that way ...

**Comment by l0rd** [comment]
Fixed, thanks for the PR

**Pull request closed without merging**